pettingzoo/classic/chess/chess_utils.py

```diff
@@ -281,8 +284,8 @@ def get_observation(orig_board, player):
 
         """
     base = BASE
-    OURS = 0
-    THEIRS = 1
+    OURS = 1
```

Comment:
Maybe make a comment on this as well? I'm not 100% sure what this change does, and to a dev unfamiliar looking at this in the future they might be like hmm this PR seems to switch and flip a lot of different things, but I have no idea which line corresponds to which change exactly

Reply:
I write down the reason why I change the flag `OURS` and `THEIRS` in `chess_utils.py` as well as this pull request.

Reply:
I swap the value of `OURS` and `THEIRS` because the module `chess` defined `chess. WHITE=1` and `chess.BLACK=0`. I guess it came from a typo because in `pettingzoo.chess_v6`, `player_0` play the white.

Reply:
Ok this is good